types/tape/tape-tests.ts

```diff
@@ -1,5 +1,7 @@
 import tape = require('tape');
 
+import Test = require('tape/lib/test');;
```

Comment:
the linter is crashing because it didn't expect the double semicolon. I'll file a bug on `@definitelytyped/header-parser`

Reply:
oh whoops, thanks.

Reply:
Note that it also tripped up on the unnecessary semicolon i had after the `}` in `declare module 'blah' {};`